### DISCUSSION TITLE
Having a error with loading the map ((((Error getting blips! Maybe the server is down or, the config is setup incorrectly.))))

### DISCUSSION THREAD

**Discussion by WolfoTheWolf** [TGRHavoc/live_map#58] [Q&A] [answered]
So I have just re-installed the live map and cleared my server cache now im having a issue actually connecting to it (btw for this demo im using xxamp) Set it up how I did last time nothing different now it does not want to show blips (when I added it I was getting a node error so I did `npm i` in the cmd of the live_map folder to install node winch made the error disappear) Just cant figure out why its saying there is a error. 
![Capture](https://user-images.githubusercontent.com/56661068/116833736-e91fa800-ab88-11eb-95a9-43ca77af7d5e.PNG)
![Captu2re](https://user-images.githubusercontent.com/56661068/116833738-e91fa800-ab88-11eb-8e93-6f64f88a7c3f.PNG)

**Comment by TGRHavoc** [comment]
Generally this is caused by a miss-configuration somewhere.

If your website using SSL (https) then you have to set up a reverse proxy to the resource for it to work.

Double check the values are correct.
You should be able to navigate to `http://IP:socket_port/blips.json` to see if the resource is working as intended and serving the blip file.
It _could_ also be your firewall so make sure that the socket port is allowed through it.

**Comment by WolfoTheWolf** [comment]
So I done that and I am getting blip data back
![Capture2](https://user-images.githubusercontent.com/56661068/116934067-73701680-ac32-11eb-925f-5387a8a2fbe5.PNG)
![Capture](https://user-images.githubusercontent.com/56661068/116934071-7408ad00-ac32-11eb-9fac-2aa605319558.PNG)

**Comment by TGRHavoc** [comment]
What errors are you getting in the browser?

You should be able to press `Ctrl + Shift + I` to open the developer tools in your browser. Then click "Console" and see if there's errors there (the errors in regards to the images are normal).

**Comment by WolfoTheWolf** [comment]
Here is the error

![Captur444e](https://user-images.githubusercontent.com/56661068/117053159-a07e0100-ace6-11eb-8d14-72c8e9281502.PNG)

**Comment by WolfoTheWolf** [comment]
Disregard, I believe it is because it was specified as `HTTPS` I changed it to `HPPT` and it solved the issue, sorry about that

**Comment by TGRHavoc** [comment, marked as answer]
Yeah, it is.

You _can_ use HTTPs but, it does require setting up a "reverse proxy" on your webserver.
